changes/ocd-3644.sql

```diff
@@ -0,0 +1,24 @@
+alter table openchpl.certification_criterion_attribute
+drop column if exists service_base_url_list;
+
+alter table openchpl.certification_criterion_attribute
+add column service_base_url_list bool not null default false;
+
+alter table openchpl.certification_result
+drop column if exists service_base_url_list cascade;
+
+alter table openchpl.certification_result
+add column service_base_url_list varchar(1024);
+
+alter table openchpl.pending_certification_result
+drop column if exists service_base_url_list;
+
+alter table openchpl.pending_certification_result
```

Comment:
definitely need to make it "text" here otherwise we get unexpected exceptions when the user puts some giant chunk of text here that exceeds the length (has happened with other stringy fields)